src/coreclr/src/debug/di/rsthread.cpp

```diff
@@ -2788,8 +2790,6 @@ CordbUnmanagedThread::CordbUnmanagedThread(CordbProcess *pProcess, DWORD dwThrea
 #ifdef TARGET_X86
     m_pSavedLeafSeh(NULL),
 #endif
-    m_stackBase(0),
-    m_stackLimit(0),
     m_continueCountCached(0)
```

Comment:
how is the initialization order determined for these? Is it based on how they are defined?

Reply:
Yes, the warning required an init order to match the definition order.